framework/src/wifi_manager/wifi_manager.c

```diff
@@ -490,7 +490,7 @@ wifi_manager_result_e wifi_manager_set_mode(wifi_manager_mode_e mode, wifi_manag
 		}
 
 		g_manager_info.mode = SOFTAP_MODE;
-		strncpy(g_manager_info.ssid, config->ssid, softap_config.ssid_length);
+		strncpy(g_manager_info.ssid, config->ssid, 32);
 		wifi_status_set(CLIENT_DISCONNECTED);
 		wifi_mutex_release(w_mutex);
 
```

Comment:
can you please state the issue with usage of strcpy

Reply:
strcpy is vulnerable. See the message below:
Use of vulnerable function 'strcpy' at wifi_manager.c. This function is unsafe, use strncpy instead.